src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  background-color: #fff;
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+}
+
+.logo {
+  display: block;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  line-height: 60px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  margin-left: -20px;
+  list-style: none;
+  padding: 0;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  display: inline-block;
+  height: 60px;
+  margin-left: 20px;
+  color: black;
+  text-decoration: none;
+  position: relative;
+}
+
+.is-active {
+  color: var(--blue-color);
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: var(--blue-color);
+}
+
+.nav__link:hover {
+  color: var(--blue-color);
+}
+
+.nav__link:hover::after {
+  width: 100%;
+  transition: width 0.3s ease;
```

Comment:
The transition should be applied to the initial state of the element rather than the `:hover` state to ensure smooth transitions. Consider moving `transition: width 0.3s ease;` to the `.nav__link::after` selector.